assets/src/css/admin/setup.scss

```diff
@@ -0,0 +1,135 @@
+/**
+ * Give Setup SCSS
+ *
+ * @description Setup Page Styles
+ * @package     Give
+ * @subpackage  SCSS/Admin
+ * @copyright   Copyright (c) 2020, GiveWP
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+*/
+
+@import url(https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap);
```

Comment:
In other areas of GiveWP we are enqueuing google fonts as a separate style, which is used as a dependency. Curious if you have a strong opinion on us moving away from that norm, otherwise I think it may be best to be consistent in how we load google fonts.